src/js/stories/DropdownStories.jsx

```diff
@@ -13,48 +15,114 @@ const style = {
     flexDirection: 'column',
     justifyContent: 'space-around',
     alignItems: 'center',
-    height: 400,
     padding: 10,
 };
 
-function createOptions(count) {
+const a2z = 'ABCDEFGHIJKLM NOPQRSTUVWXYZ';
+
+function createOptions(text = 'Option', asObject = false) {
     const options = [];
-    options.push('ABCDEFGHIJKLMNOPQRSTUVWXYZ');
-    for (let index = 0; index < count; index++) {
-        options.push(`Option ${options.length + 1}`);
+
+    for (let index = 0; index < 200; index++) {
+        const label = `${text} ${options.length + 1}`;
+        options.push(!asObject ? label : { label });
     }
     return options;
 }
 
+
+
 function Default() {
+    Utils.createCssRule(
```

Comment:
Fixed in f7fda7e. I cleaned up this method a bit to avoid creating duplicate rules which happens by default (presumably due to some webpack loading oddities). Only catch is that if you remove a rule you need to do a full page refresh to get rid of it; otherwise it seems to work pretty well in all browsers.